bin/start-server-micros.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+cd "$(dirname "$0")/.."
+
+if [ "$MICROS_GROUP" = "WebServer" ]; then
+  COMMAND="start:main:production"
+fi
+
+if [ "$MICROS_GROUP" = "Worker" ]; then
+  COMMAND="start:worker:production"
+fi
+
+if [ -z "$COMMAND" ]; then
+  echo "Wrong MICROS_GROUP environment parameter: ${MICROS_GROUP}"
+  exit 1
+fi
```

Comment:
Wouldn't it make more sense to use case statements?

Reply:
Good point. Didn't know bashscript has it